source/TimePicker.js

```diff
@@ -470,10 +487,11 @@
 					if (this.meridiemEnable === true) {
 						this.createComponent(
 							{classes: 'moon-date-picker-wrap', components:[
-								{kind:'moon.MeridiemPicker', name:'meridiem', classes:'moon-date-picker-field', value: valueHours > 12 ? 1 : 0, meridiems: this.meridiems || ['am','pm'], onChange: 'meridiemPickerChanged'},
+								{kind:'moon.MeridiemPicker', name:'meridiem', classes:'moon-date-picker-field', meridiems: this.meridiems || [{name: 'AM', start: '00:00', end: '11:59'}, {name: 'PM', start: '12:00', end: '23:59'}], max: this.meridiems? this.meridiems.length - 1 : 1, onChange: 'meridiemPickerChanged'},
 								{name: 'meridiemLabel', content: this.meridiemText, classes: 'moon-date-picker-label moon-divider-text'}
 							]}
 						);
+						this.$.meridiem.value = this.getMeridiemIndex(values.hour, values.minute);
```

Comment:
I was thinking more like an `updateValue` method we expose on `MeridiemPicker`, that would accept an `hours` and a `minutes` parameter, and would encapsulate the logic found in `getMeridiemIndex`. I realize it's difficult to properly separate everything without doing a wholesale change, but I would like to reduce the amount of meridiem knowledge the `TimePicker` has, and keep that within `MeridiemPicker`. What do you think about this?